src/components/[guild]/EditGuild/hooks/useEditGuild.ts

```diff
@@ -250,31 +261,55 @@ const useEditGuild = ({ onSuccess, guildId }: Props = {}) => {
         })
       } else {
         if (admin.creations.failedCount > 0) {
-          showErrorToast("Failed to create some admins")
+          showErrorToast({
+            error: "Failed to create some admins",
+            correlationId: admin.creations.correlationId,
+          })
         }
         if (admin.deletions.failedCount > 0) {
-          showErrorToast("Failed to delete some admins")
+          showErrorToast({
+            error: "Failed to delete some admins",
+            correlationId: admin.deletions.correlationId,
+          })
         }
 
         if (contacts.creations.failedCount > 0) {
-          showErrorToast("Failed to create some contacts")
+          showErrorToast({
+            error: "Failed to create some contacts",
+            correlationId: contacts.creations.correlationId,
+          })
         }
         if (contacts.updates.failedCount > 0) {
-          showErrorToast("Failed to update some contacts")
+          showErrorToast({
+            error: "Failed to update some contacts",
+            correlationId: contacts.updates.correlationId,
+          })
         }
         if (contacts.deletions.failedCount > 0) {
-          showErrorToast("Failed to delete some contacts")
+          showErrorToast({
+            error: "Failed to delete some contacts",
+            correlationId: contacts.deletions.correlationId,
+          })
         }
 
         if (featureFlags.creations.failedCount > 0) {
-          showErrorToast("Failed to create some feature flags")
+          showErrorToast({
+            error: "Failed to create some feature flags",
+            correlationId: featureFlags.creations.correlationId,
+          })
         }
         if (featureFlags.deletions.failedCount > 0) {
-          showErrorToast("Failed to delete some feature flags")
+          showErrorToast({
+            error: "Failed to delete some feature flags",
+            correlationId: featureFlags.deletions.correlationId,
+          })
         }
 
-        if (guildUpdateResult === null) {
-          showErrorToast("Failed to update guild data")
+        if (guildUpdateResult.error) {
```

Comment:
I think we need a `?` here, because it can be `undefined`
```suggestion
        if (guildUpdateResult?.error) {
```